ple/games/donkeykong/Fireball.py

```diff
@@ -0,0 +1,116 @@
+__author__ = 'Erilyth'
+import pygame
+import random
+import math
+from OnBoard import OnBoard
+
+'''
+This class defines all our fireballs.
+A fireball inherits from the OnBoard class since we will use it as an inanimate object on our board.
+Each fireball can check for collisions in order to decide when to turn and when they hit a player.
+'''
+
+class Fireball(OnBoard):
+    def __init__(self, raw_image, position, index,speed):
+        super(Fireball,self).__init__(raw_image, position)
+        #Set the fireball direction randomly
+        self.__direction = int(math.floor(random.random() * 100)) % 2
```

Comment:
same as the above player class, this should take a rng object and call `self.rng.rand()`